MIGRATION.md

```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
 />;
 ```
 
+## DateTime
+
+The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
+
+```tsx static=true
+import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2020-08-17T15:45:00" />
+<DateTime date="2020-08-17T15:45:00" format="day" />
+<DateTime date="2020-08-17T15:45:00" format="weekday" />
+<DateTime date="2020-08-17T15:45:00" format="time" />
+```
+
+### How to migrate your DateTime components
+
+Update the `format` value and import. It changes from this:
+
+```tsx static=true
+import { DateTime } from '@contentful/forma-36-react-components';
+
+<DateTime date="2020-08-17T15:45:00" format="WEEKDAY_DATE" />;
+```
+
+to this:
+
+```tsx static=true
+import { DateTime } from '@contentful/f36-components';
+
+<DateTime date="2020-08-17T15:45:00" format="weekday" />;
+```
+
+## RelativeDateTime
+
+The API of the RelativeDateComponent was not changed.
+
+### How to migrate your RelativeDateTime components
+
+You just need to update import. From this:
```

Comment:
For consistency throughout the guide, should we phrase this as follows?

To migrate RelativeDateComponent to v4, you must update the import. From this: